src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx

```diff
@@ -77,16 +77,6 @@ describe("Component - Curriculum Header", () => {
   });
 
   test("user can see the tabular navigation", async () => {
-    const { findByTestId } = renderComponent();
-    const tabularNav = await findByTestId("tabularNav");
-    expect(tabularNav).toBeInTheDocument();
-    const links = await findAllByRole(tabularNav, "link");
-    expect(links).toHaveLength(2);
-    expect(links[0]).toHaveTextContent("Unit sequence");
-    expect(links[1]).toHaveTextContent("Overview");
-  });
-
-  test("user can see the tabular navigation (downloads prerelease)", async () => {
     // NOTE: This is only active during testing.
     mockPrerelease("curriculum.downloads");
```

Comment:
Do we still need to mock the pre-release?

Reply:
Yeah I think we can use it for now for the visualiser cycle 2 stuff, until we get proper flags implemented.